currency/rate_engines.go

```diff
@@ -0,0 +1,42 @@
+package currency
+
+import "strings"
+
+const (
+	CustomRates = iota
+	PBSRates
+)
+
+type RateEngines struct {
+	conversions []Conversions
```

Comment:
Consider having `customRates` and `serverRates` fields instead of using the slice. I don't see why `re.conversions[CustomRates]` wouldn't be clearer as `re.customRates`.

Reply:
Good point. Done